src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingRemoteHostClientWrapper.cs

```diff
@@ -5,6 +5,8 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.Remote;
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
💡 It would be better to mark the methods as obsolete and let downstream consumers suppress this warning.

Reply:
We do not have a replacement for LUT yet. We shouldn't use TryCreateSessionAsync in Roslyn though.